### ISSUE TITLE
Rate limiting from brute force protection prevents login with the Android client

### ISSUE THREAD

**Issue by willsALMANJ** [nextcloud/android#460]
### Actual behaviour
If you enter the wrong password too many times, brute force protection makes it impossible to log in with the Android client because the client seems to time out after 10 seconds while the brute force protection enforces a 30 second delay.

### Expected behaviour
The Android client's timeout should be long enough to handle the delay from brute force protection. Perhaps after waiting 10 seconds the client could display a message saying that login is slow and the current IP address might be blacklisted.
 
### Steps to reproduce
1. Enter the wrong password enough times to get your IP address rate limited by brute force protection.
2. Try to log in with the Android client
3. Receive a "Server took too long to respond" message after 10 seconds.

I hope I have diagnosed this situation correctly. I changed my password on the server and then tried to connect a few times from the Android client with the old password before remembering that it had changed. After that, I could not log in again even after ensuring the password was correct until I had cleared my ip address from the brute force blacklist as described here: https://help.nextcloud.com/t/how-can-i-unblock-an-ip-blocked-through-brute-force-detection/5731

### Environment data
Android version: 6.0.1

Device model: Motorola Moto E

Stock or customized system: CyanogenMod

Nextcloud app version: 1.4.0 rc2 (latest version on F-Droid at this time)

Nextcloud server version: 10.0.2

PR: https://github.com/nextcloud/android/pull/1099

**Comment by linucksrox** [comment]
We experienced this issue today with the Windows desktop client for a couple users (not sure if that should be a separate issue, or if this issue should be broadened to include other clients).

**Comment by tobiasKaminsky** [comment]
There is a  separate repository for windows client so please report this issue also over there.

**Comment by tobiasKaminsky** [comment]
Happened to me also on my test system, very annoying as the message is technically correct, but does not help to solve the problem.
@LukasReschke is there some response we can get, e.g. too many login attempts, from server?

**Comment by xXSTrikeXx** [comment]
any updates? pr exists, but even open :D

**Comment by willsALMANJ** [comment]
What is the relevant PR? Nothing is linked to this issue.

**Comment by tobiasKaminsky** [comment]
PR #1099, it is mentioned on the bottom of the starting issue.
